src/workloads/query/AggregationsOutput.yml

```diff
@@ -0,0 +1,520 @@
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput6]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput7]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput8]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput9]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput10]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput11]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput12]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput13]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput14]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput15]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput16]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput17]}}
+            key: {shardKey: "hashed"}
+        - OperationName: AdminCommand
+          OperationCommand:
+            shardCollection: {^PreprocessorFormatString: {format: "test.%s", withArgs: [*ShardedCollOutput18]}}
+            key: {shardKey: "hashed"}
+# TODO: Make a lot of sharded collections
```

Comment:
Leftover?

Reply:
Sorry, I forgot to remove this when creating all the shardedColl stuff